core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1207,7 +1207,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_BACKUP_DIRECTORY =
       new Builder(Name.MASTER_BACKUP_DIRECTORY)
-          .setDefaultValue(String.format("${%s}/alluxio_backups", Name.WORK_DIR))
+          .setDefaultValue("/alluxio_backups")
```

Comment:
should this be `String.format("${%s}/alluxio_backups", Property.MASTER_MOUNT_TABLE_ROOT_UFS. getDefaultValue()`?

Reply:
We want to back up to the root ufs.
Like  root ufs maybe is `hdfs://host:port/alluxio/`
The backup will backup to `hdfs://host:port/alluxio_backups/`

Reply:
The backup folder should not under alluxio userspace ideally. We don't want users to use alluxio API to manage their backups

Reply:
ack